src/test/resources/application-test.yml

```diff
@@ -18,3 +18,4 @@ caserefgeneratorkey: rPr3qsjlU42hsDX5npfuCzlyF4UYfDTO
 queueconfig:
   consumers: 2 #this should be more than adequate to check for any multithreading problems
   retry-delay: 10  #this is for the tests around multiple receipts for the same case locking
+  rh-uac-queue: events.rh.uacUpdate
```

Comment:
this needs to be deleted. Only thing that goes in this file is stuff to point at a different Rabbit/Postgres